Fabric.Platform.Auth/AuthorizationMiddleware.cs

```diff
@@ -33,5 +34,28 @@ public static AppFunc Inject(AppFunc next, string[] requiredScopes, string[] all
                 return Task.CompletedTask;
             };
         }
+
+        private static bool HasPath(string[] allowedPaths, string pathToCheck)
+        {
+            foreach(var path in allowedPaths)
+            {
```

Comment:
Maybe move the asterisk to a constant?

Reply:
PathSymbol.AnyNumberOfCharacters, I did that.